SvrfSDK/Source/SvrfSDK.swift

```diff
@@ -222,14 +228,50 @@ public class SvrfSDK: NSObject {
                 svrfRequest.state = .completed
 
                 if let failure = failure, var svrfError = error as? SvrfError {
-                    svrfError.svrfDescription = SvrfErrorDescription.response.rawValue
+                    if svrfError.svrfDescription == nil {
+                        svrfError.svrfDescription = SvrfErrorDescription.response.rawValue
+                    }
                     failure(svrfError)
                 }
 
             })
         }
     }
 
+    /**
+    Requests data from the Svrf by endpoint (f.e. "/vr/search")
```

Comment:
```suggestion
    Requests data from the Svrf by endpoint (e.g. `/vr/search`)
```

e.g. not f.e.  It’s an abbreviation for the Latin phrase “exempli gratia” which means “for example”. So the user does not have to include the API version?